gatsby-site/gatsby-node.js

```diff
@@ -83,61 +107,60 @@ exports.sourceNodes = async helpers => {
   /** @type {HubhubPayload} */
   const data = await response.json()
 
-  const hub = data.map_data.find(hub => hub.id === HUB_ID)
-
-  if (!hub) {
-    reporter.panicOnBuild(
-      `Hub ${HUB_ID} does not exist in Hubhub. Please check your config, or else contact a Hubhub admin.`
-    )
-  }
-
-  const hubNodeData = {
-    name: parseName(hub.name),
-    about: hub.about,
-    email: hub.email,
-    website: hub.website,
-    facebook: hub.facebook,
-    instagram: hub.instagram,
-    twitter: hub.twitter,
-  }
-
-  const hubNodeLinks = {
-    hero___NODE:
-      !hub.hero_image || hub.hero_image.length === 0
-        ? null
-        : await fileNodeFromRemoteFile(helpers, hub.hero_image[0]),
-    logo___NODE:
-      !hub.logo_image || hub.logo_image.length === 0
-        ? null
-        : await fileNodeFromRemoteFile(helpers, hub.logo_image[0]),
-    documents___NODE: !hub.documents
-      ? []
-      : await Promise.all(
-          hub.documents.map(remoteFile =>
-            fileNodeFromRemoteFile(helpers, remoteFile)
-          )
-        ),
-    images___NODE: !hub.images
-      ? []
-      : await Promise.all(
-          hub.images.map(remoteFile =>
-            fileNodeFromRemoteFile(helpers, remoteFile)
-          )
-        ),
-  }
-
-  createNode({
-    id: createNodeId(`Hub-${hub.id}`),
-    ...hubNodeData,
-    ...hubNodeLinks,
-    internal: {
-      type: "Hub",
-      contentDigest: createContentDigest({
+  await Promise.all(
+    data.map_data.map(async hub => {
+      const name = parseName(hub.name)
+
+      const hubNodeData = {
+        name,
+        slug: slugFromName(name),
```

Comment:
Note: this is only a temporary implementation until hubhub has a field for the slug